### PR TITLE
SEAR-277-Spinning Search Page on Certain Version of IE

### PR DESCRIPTION
**Pull request by Nasir2712**
#### Which User story does this relate to (link)?
-https://osi-cwds.atlassian.net/browse/SEAR-277

#### Brief feature description
- fix spinner on ie

#### Does this PR block/effect other stories?
- n/a

#### Do any commands needs to be run in order to run the project with this PR?
-n/a

#### Code coverage percent diff as a result of this PR?
-no

### CODE REVIEW

**Comment by sunilb0575** [comment]
why we are not using React-Spinkit?

File: app/assets/stylesheets/facility.scss [hunk 1]
```diff
@@ -177,6 +177,127 @@
   content: "\f110";
   font-family: FontAwesome;
   font-size:30px;
-  position: absolute;
   top: 0; 
 }
+
+.sk-circle {
```
Comment (ratneshraval) [inline app/assets/stylesheets/facility.scss]: This should be separate css

Comment (Nasir2712) [reply app/assets/stylesheets/facility.scss]: ok

**Comment by Nasir2712** [comment]
jenkins, retest this please

**Comment by Nasir2712** [comment]
jenkins, retest this please

**Comment by Nasir2712** [comment]
jenkins, retest this please

**Pull request merged**